### DISCUSSION TITLE
hfs v3 0.43 columns in the access log and the error log

### DISCUSSION THREAD

**Discussion by gogi100** [rejetto/hfs#208] [General]
can you add columns in the access log and the error log, because of i use those logs in my log server

**Comment by gogi100** [comment]
can the location of the logs be changed? since a huge number of files are created because I have a lot of logins to the system, can the logs be put on another partition separate from the hfs location?

**Comment by rejetto** [comment]
What columns are you missing?

**Reply by gogi100** [reply]
sorry, my mistake, I didn't see the date and time in the log.
does the log format meet the standards for log servers, i have the kiwi syslog?
can the location of the logs be changed? since a huge number of files are created because I have a lot of logins to the system, can the logs be put on another partition separate from the hfs location?

**Comment by rejetto** [comment]
the log format is Apache standard.
choose the location of logs in the Options page.

**Reply by gogi100** [reply]
what path should I put the log in the admin panel/options, if I want the log to be on a different partition than the hfs folder

**Comment by rejetto** [comment]
you have to enter the full path to the file, wherever you want it
